### ISSUE TITLE
Server crash after restart resource

### ISSUE THREAD

**Issue by mrgharabaghi** [altmp/altv-issues#1562]
### Description of the problem

It's better to look at `Reproduction steps`.

### Reproduction steps

1. Start the server with a normal `bat` file with `altv-server.exe` in it.
2. Restart `res2` with `restart folder1/res2`.
3. Restart `res1` with `restart res1`.
4. The server will crash.

### Expected behaviour

Should not crash.

### Additional context

`server.cfg`
```
resources: [
    res1
    folder1/res2
]
```

### Operating system

Windows 10

### Version

release/11.3

**Comment by mrgharabaghi** [comment]
If there is some technical issue with `--extra-res-folder` please let us know, otherwise, this key is very handy and should not remove.

**Comment by mrgharabaghi** [comment]
According to Leon, it makes sense that the `--extra-res-folder` is not removed.
https://discord.com/channels/371265202378899476/371271006360436757/997927610552168550

**Comment by mrgharabaghi** [comment]
With version 12.5 (release), the server crashes with the normal resource path.
This happens with `debug = false` and `debug = true`.

1. Start the server with a normal `bat` file with `altv-server.exe` in it.
2. Restart `res` with `restart res`.
3. The server will crash.

`server.cfg`
```
resources: [
    'res'
]
```
`crashdump`
- [additional.txt](https://github.com/altmp/altv-issues/files/9210153/additional.txt)
- [client.log](https://github.com/altmp/altv-issues/files/9210163/client.log)

**Comment by deluvas1911** [comment]
Crash can be reproduced on current verison

**Comment by C0kkie** [comment]
Will be fixed in release/12.9
